snippets/language-html.cson

```diff
@@ -92,13 +92,13 @@
     'body': '<dfn>$1</dfn>$0'
   'Description List':
     'prefix': 'dl'
-    'body': '<dl id="$1">\n\t$2\n</dl>'
+    'body': '<dl class="$1">\n\t$2\n</dl>'
   'Definition Term':
     'prefix': 'dt'
     'body': '<dt>$1</dt>$0'
   'Div':
     'prefix': 'div'
-    'body': '<div${1: id="${2:name}"}>\n\t$3\n</div>'
+    'body': '<div${1: class="$2"}>\n\t$3\n</div>'
```

Comment:
Yeah, I'd say add the space and remove the outer marker, that makes it behave like the others, if people really need a plain div, typing it out manually is pretty quick.